src/Sylius/Bundle/ApiBundle/DataProvider/Helpers/ProductDataProviderHelper.php

```diff
@@ -18,8 +18,13 @@ public static function setCustomPropertiesToProduct($product, $channelCode)
         $prices = [];
 
         foreach ($product->getVariants()->getValues() as $i => $variant) {
-            $prices[] = $variant->getChannelPricings()[$channelCode]->getPrice();
-            $onHand += $variant->getOnHand();
+            if (isset($variant) && ($variant instanceof ProductVariantInterface)) {
```

Comment:
strange, on my local looks different

Reply:
> @brizyNicu Important: make a migrations: because database schema isn't up to date in your PR https://i.imgur.com/8HGAqQu.png

Migrations are already made just run them